protoc-artifacts/README.md

```diff
@@ -54,7 +54,62 @@ Before you can upload artifacts to Maven Central repository, make sure you have
 read [this page](http://central.sonatype.org/pages/apache-maven.html) on how to
 configure GPG and Sonatype account.
 
-Use the following command to upload artifacts:
+You need to perform the deployment for every platform that you want to
+suppport. DO NOT close the staging repository until you have done the
```

Comment:
Just in case anyone wants to try to remove that dependency, I came across this:
https://github.com/wkhtmltopdf/wkhtmltopdf/commit/64e3f87ea05303c6a0331c8b13eeeb92f60e3b3c

We may just need -static flag.